src/sentry/ingest/consumer/processors.py

```diff
@@ -188,12 +188,8 @@ def process_event(
             )
         elif data.get("type") == "feedback":
             if features.has("organizations:user-feedback-ingest", project.organization, actor=None):
-                save_event_feedback.delay(
```

Comment:
Let's delete the save_event_feedback function.

Reply:
discussed on a call:
- wait until feedback consumer is deployed (to prod?) and actually consuming all the feedbacks, before merging this change
- same with deleting save_event_feedback

Reply:
You can delete save_event_feedback on this pull.  We're not going to merge it until we're ready so its cool.

Reply:
+1